src/components/VideoThumbnail/VideoThumbnail.scss

```diff
@@ -24,6 +24,7 @@ $progress-bar-height: 6px;
 }
 
 .PlayButton {
+  --pc-play-button-focused-state-overlay: rgba(223, 227, 232, 0.3);
```

Comment:
This rgba value is an extended state color that used to be part of [color-palette-data](https://github.com/Shopify/polaris-react/pull/4715/commits/2508ebb7e23cc82748edf7d841d889d72fc1ea90). I didn't want to create a one off value so I put it into a `--pc-*` token. Would love feedback on this approach.

Reply:
This is the right call for now. We can in another PR go through all the hard coded color values.